bridge/dd_wrap_autoloader.php

```diff
@@ -1,31 +0,0 @@
-<?php
-
```

Comment:
I imagine that at some point we've told people to set this ini manually, such as people installing from `pecl`. To help them upgrade I think we should emit a diagnostic or something and then include `dd_init.php`. What do you think?

Reply:
+1 to @morrisonlevi's comment. There are also many people manually including this file from their own bootstrap files. I would change this file to `include __DIR__ . '/dd_init.php';` and leave a comment mentioning that this file was deprecated in 0.46.0.

Reply:
love it